hibernate-core/src/main/java/org/hibernate/tool/schema/SourceType.java

```diff
@@ -26,8 +26,8 @@
 	/**
 	 * "script" - External DDL script(s) are used as the exclusive source for generation.  The scripts for schema
 	 * creation and dropping come from different sources.  The creation DDL script is identified by the
-	 * {@value AvailableSettings#HBM2DDL_CREATE_SCRIPT_SOURCE} setting; the drop DDL script is identified by the
```

Comment:
Again, `@value` should have worked here.

Reply:
Which command did you use, btw? It would be too good to have it work.

Reply:
IMO we should report this to the Gradle team and see what they say.  `{@value}` is a perfectly legal inline doclet - https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#value.

I'd revert the changes related to this and let's wait to hear back from them.

Reply:
I finally solved the issue by adding package to the class in `@value` reference. Not sure why but it seems there is some subtle difference between `@value` and `@see`.

Reply:
> I finally solved the issue by adding package to the class in `@value` reference. Not sure why but it seems there is some subtle difference between `@value` and `@see`. Also, 
https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#value dictates the presence of package before class name as well.